Source/Core/DolphinQt/Settings/GameCubePane.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 Dolphin Emulator Project
+// Copyright 2019 Dolphin Emulator Project
```

Comment:
No need to update the copyright notice, we keep the year as creation rather than last updated.